app/helpers/exercise_helper.rb

```diff
@@ -2,4 +2,17 @@ module ExerciseHelper
   def embedding_parameters(exercise)
     "locale=#{I18n.locale}&token=#{exercise.token}"
   end
+
+  def qa_js_tag
+    javascript_include_tag qa_url + "/assets/qa_api.js"
```

Comment:
This might break in prod env cause this file cannot be called like this. CodePilot need to asure that this non fingerprinted URL is handled the right way.